docs/content/concepts/configuration/config-schema-legacy.mdx

```diff
@@ -0,0 +1,278 @@
+---
+
+# Run Configuration (Legacy)
+
+<Note>
+  This guide covers using legacy APIs for the Dagster config system. For docs on
+  the new Pythonic config system introduced in Dagster 1.3, see the{" "}
+  <a href="/concepts/configuration/config-schema">
+    updated configuration guide
+  </a>
+  .
+</Note>
+
+Run configuration allows providing parameters to jobs at the time they're executed.
+
+It's often useful to provide user-chosen values to Dagster jobs or software-defined assets at runtime. For example, you might want to choose what dataset an op runs against, or provide a connection URL for a database resource. Dagster exposes this functionality through a configuration API.
+
+Various Dagster entities (ops, assets, resources) can be individually configured. When launching a job that executes (ops), materializes (assets), or instantiates (resources) a configurable entity, you can provide _run configuration_ for each entity. Within the function that defines the entity, you can access the passed-in configuration off of the `context`. Typically, the provided run configuration values correspond to a _configuration schema_ attached to the op/asset/resource definition. Dagster validates the run configuration against the schema and proceeds only if validation is successful.
+
+A common use of configuration is for a [schedule](/concepts/partitions-schedules-sensors/schedules) or [sensor](/concepts/partitions-schedules-sensors/schedules) to provide configuration to the job run it is launching. For example, a daily schedule might provide the day it's running on to one of the ops as a config value, and that op might use that config value to decide what day's data to read.
+
+---
+
+## Relevant APIs
+
+| Name                                                 | Description                                              |
+| ---------------------------------------------------- | -------------------------------------------------------- |
+| <PyObject module="dagster" object="ConfigSchema"  /> | See details with code examples in the API documentation. |
+
+---
+
+## Defining and accessing configuration for an op, asset, or resource
+
+Configurable parameters accepted by an op, asset, or resource are specified by providing a `config_schema` to the corresponding decorator. The structure of a `config_schema` is flexible and [fully documented in the API Reference](/\_apidocs/config). However, most of the time you will want to provide a Python dictionary, with keys the names of parameters and values the types of those parameters.
+
+During execution, the specified parameters are accessible within the body of the op/asset/resource under `context.op_config` (for ops/assets) or `context.resource_config` (for resources). It might seem confusing that asset config is accessed under `context.op_config` instead of `context.asset_config`. However, assets are wrappers for ops, so when we access asset config we are literally just accessing config for the underlying op.
```

Comment:
> However, assets are wrappers for ops, so when we access asset config we are literally just accessing config for the underlying op.
 
"assets are wrappers for ops" isn't really accurate. What is more accurate is that an `@asset` decorated function is a single asset that is materialized with a single op and the function is the body of that underlying op. This contrasts with an `AssetsDefinition` where a number of assets could be backed by a single op (dbt being a notable example). Curious what @sryza 's preferred language is here.

Reply:
One of the ways we've phrased this is: "Under the hood, every software-defined asset contains an op (or graph of ops), which is the function that’s invoked to compute its contents."

https://docs.dagster.io/guides/dagster/how-assets-relate-to-ops-and-graphs#software-defined-assets